readme.txt

```diff
@@ -2,6 +2,9 @@
 
 == Changelog ==
 
+= [] TBD =
```

Comment:
Hey guys would you mind correct me here, what the version should be added here.

Thanks.

Reply:
` = [4.7.6] TBD =`